doc/api/cli.md

```diff
@@ -848,6 +848,13 @@ Enables report to be generated on uncaught exceptions. Useful when inspecting
 the JavaScript stack in conjunction with native stack and other runtime
 environment data.
 
+### `--suppress-warnings`
+<!-- YAML
+added: REPLACEME
+-->
+
+Silence deprecation and experimental warnings for specified codes.
```

Comment:
What's the difference between this and the existing --no-warnings flag?

Reply:
@jasnell - I have introduced the new flag `--suppress-warnings` and also introduced codes for experimental warnings which enables this flag to silence the experimental and deprecation  warnings for supplied codes. The idea is to support silencing warnings for experimental and deprecation warnings and enabled the flag to take multiple codes which is not the case with existing `--no-warnings`

Reply:
Yes, and what I'm saying is that the `--no-warnings` flag can just be extended to support this without introducing a new flag. `--no-warnings` (with no arguments) would continue to work as it does today, while `--no-warnings={list of codes}` would suppress the named codes. In either case, the documentation here should indicate that the flag takes a list of codes.

Reply:
@jasnell - well instead of introducing new clip option we can extend the existing `--no-warnings` option right . yeah I understood and thanks for the review.